Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -416,6 +479,8 @@ namespace Multiplayer
     {
         AZ_PROFILE_SCOPE(MULTIPLAYER, "MultiplayerSystemComponent: OnTick");
 
+        const AZStd::chrono::utc_clock::time_point startMultiplayerTickTime = AZStd::chrono::utc_clock::now();
```

Comment:
This should be `steady_clock` not utc_clock.
The utc clock is affected by the system tiem and could go backwards if an NTP sync occurs(or the system clock is manually changed on the mahcine)

Reply:
Replaced with steady_clock